1201 email accessability

EricDube opened the pull request:
Fixes a small issue where the Accessibility field was required to click at least once

andr0272 commented on src/pages/RegistrationPage.js:
```diff
@@ -122,7 +122,8 @@ class RegistrationPage extends React.Component {
       if (windowExists()) {
-       // registrationFields.accessibility = 'accept_anything' //TODO: why?
+        //This is to force allow accessibility to not be required on first page load
+        registrationFields.accessibility = 'accept_anything'
```
There's a lot to take in on this PR. might need a bit.

EricDube commented:
> not related to this PR but to the previous one
> accessibility is an array, in the ReviewPage.js
> the translateReason function needs reason[0] === 'yes' in order to work

Added a fix for that, also added a undefined check just in case
